### DISCUSSION TITLE
Set Chart Background Color

### DISCUSSION THREAD

**Discussion by dph10** [GSI-CS-CO/chart-fx#507] [Q&A] [answered]
Hello,

How do I set the background color of XYChart to solid white?  Right now the default looks like it may be transparent or the same gray background as JavaFX.

Also, do any of your samples show us how to make our own plot coloring schemes, such as a dark theme?

**Comment by dph10** [comment]
Also, I'd  like to turn of the plot markers and hatching for my DoubleDataSeries in my chart, but I also can't find any methods to perform these actions?  I'd like to just see the lines between points, without markers or hatching on each line series.

**Comment by JulesLassara** [comment]
Hello,

Did you find how to change the background color ? I tried looking to the DefaultRenderColorScheme class but I'm still stuck.

**Reply by dph10** [reply]
Unfortunately I have not found a solution, but I will let you know if I do.

**Reply by dph10** [reply]
I believe that I figured it out!  You have to wait until the chart is displayed on-screen, and then call the following:
`  Platform.runLater(() -> {
            this.chartMainData.lookup(".chart-plot-background").setStyle("-fx-background-color: white;");
        });`

Just substitute in your color string for "white" above, as I was trying to set the background as white.

**Reply by LilRonin** [reply, marked as answer]
.StackPane{
	-fx-background:    #ffffff;
}

put it in your css my fxmlcontroller my chart are put on a stackPane
this seems to work too

.chart-plot-background {
	-fx-background: #ffffff;
} i found it on another post
